src/main/java/wolox/training/models/User.java

```diff
@@ -22,11 +20,17 @@
 import javax.persistence.Id;
 import javax.persistence.ManyToMany;
 import javax.persistence.Table;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
 import wolox.training.exception.BookAlreadyOwnedException;
 
 @Entity
 @Table(name="users")
 @ApiModel(description = "Users to be related with books.")
+@NoArgsConstructor
+@Getter
+@NonNull
 public class User {
```

Comment:
Same here about the `@NotBlank` annotation for those fields that cannot be empty